app/services/UserService.py

```diff
@@ -17,17 +24,65 @@ async def modify_user(user_id: int, model: PydanticUserModify):
     if user_to_modify is None:
         raise HTTPException(status_code=404, detail=CommonErrorMessages.USER_NOT_FOUND)
 
-    if model.mail and await UserInDB.all().filter(mail=model.mail).count() != 0:
-        raise HTTPException(status_code=409, detail=CommonErrorMessages.MAIL_ALREADY_USED)
-    
-    if model.login and await UserInDB.all().filter(login=model.login).count() != 0:
-        raise HTTPException(status_code=409, detail=CommonErrorMessages.LOGIN_ALREADY_USED)
+    if await UserInDB.get(mail=model.mail).exists():
+        raise MailAlreadyUsedException
+
+    if await UserInDB.filter(login=model.login).exists():
+        raise LoginAlreadyUsedException
 
     try:
-        user_to_modify.update_from_dict(model.model_dump(exclude={"password", "password_confirm"}, exclude_none=True)) # type: ignore
+        user_to_modify.update_from_dict(model.model_dump(exclude={"password", "password_confirm"}, exclude_none=True))
         if model.password is not None:
             user_to_modify.hash = UserInDB.get_password_hash(model.password)
         await user_to_modify.save()
 
     except ValueError as e:
         raise HTTPException(status_code=409, detail=str(e)) from e
+
+
+async def create_user(model: PydanticUserCreate) -> str:
+    """
```

Comment:
Specify in the doctring the format of the pasword returned.
Also, please try to add a password confirmation mechanism in case the admin sets up the password.